src/playback/internal/au3/au3player.cpp

```
@@ -36,11 +36,21 @@ Au3Player::Au3Player(const muse::modularity::ContextPtr& ctx)
 
         if (st != PlaybackStatus::Stopped) {
             m_timer.start();
-        } else {
+        } else if (!AudioIO::Get()->IsStreamActive()) {
             m_timer.stop();
```

Cppcheck (2.20.0)

[style] The function 'au3VolumeToLocal' is never used.

(unusedFunction)